### DISCUSSION TITLE
How can we localize post dates in blogs?

### DISCUSSION THREAD

**Discussion by GasimGasimzada** [reactjs/reactjs.org-translation#248] [Team Posts]
I am trying to translate MarkdownPage component and currently date field is displayed in English (e.g February 23, 2019). How can we localize this field? Should I touch `onCreateNode.js` file for the proper value. Currently, the value is `date.toJSON()`, which I assume returns US format for the date.

**Comment by jcserracampos** [comment]
We, at pt-BR team, just type the correct date.
